backend/src/main/java/atlantbh/restaurants/controllers/UserController.java

```diff
@@ -0,0 +1,27 @@
+package atlantbh.restaurants.controllers;
+
+import atlantbh.restaurants.models.User;
+import atlantbh.restaurants.models.dto.UserRegistrationDTO;
+import atlantbh.restaurants.services.UserService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+public class UserController extends BaseController<User, UserService>  {
+
+    public ResponseEntity registerUser(@Valid @RequestBody UserRegistrationDTO user, BindingResult bindingResult) throws Exception {
+        if (bindingResult.hasErrors()) {
+            return ResponseEntity.badRequest().body(bindingResult.getFieldError().getDefaultMessage());
+        }
+        try {
+            User registeredUser = service.get(user);
+            return ResponseEntity.ok(registeredUser);
```

Comment:
```return ResponseEntity.ok(service.get(user));```
Since you don't use registeredUser variable anywhere else..

Reply:
Fixed in 8c841a20e489a31c66d177c8682339f9d4f19de0